libdnf/module/module_profile.cpp

```diff
@@ -0,0 +1,108 @@
+
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "libdnf/module/module_profile.hpp"
+
+#include <modulemd-2.0/modulemd.h>
+
+namespace libdnf::module {
+
+
+inline std::string gchar_to_str(const gchar * gchar_str) {
+    if (gchar_str == nullptr) {
+        return std::string();
+    }
+    return std::string(gchar_str);
+}
+
+
+std::string ModuleProfile::get_name() const {
+    if (!profile) {
+        return {};
+    }
+    return gchar_to_str(modulemd_profile_get_name(profile));
+}
+
+
+std::string ModuleProfile::get_description() const {
+    if (!profile) {
+        return {};
+    }
+    return gchar_to_str(modulemd_profile_get_description(profile, NULL));
+}
```

Comment:
This is similar to `libdnf::utils::string::c_to_str()`, but in this case it's from `gchar *` and I don't think we want to include glib in `common/utils/string.hpp`. Any advice what to do with this?

Reply:
Tbh. I have no idea why `gchar` exists as a type, it's `typedef char gchar` in `/usr/include/glib-2.0/glib/gtypes.h`. Since the typedef is not conditional in any way and therefore we should be able to rely on it actually always being `char`, I'd just drop the `gchar_to_str` function and use the `c_to_str()` that we have.

Reply:
Oh, ok, I haven't realized I can do that. I changed it to `c_to_str()`.